.ci/e2eTestingHelmDaily.groovy

```diff
@@ -45,7 +45,7 @@ pipeline {
             booleanParam(name: 'forceSkipGitChecks', value: true),
             booleanParam(name: 'forceSkipPresubmit', value: true),
             string(name: 'runTestsSuite', value: 'helm'),
-            string(name: 'SLACK_CHANNEL', value: "integrations"),
+            string(name: 'SLACK_CHANNEL', value: "infra-release-notify,integrations"),
```

Comment:
@jmlrt after this is merged, you'll receive a notification when the Helm charts failed.